### ISSUE TITLE
SmallRye: MicroProfile Metrics TCK execution fails on Java 11

### ISSUE THREAD

**Issue by rsvoboda** [smallrye/smallrye-metrics#37]
SmallRye: MicroProfile Metrics TCK execution fails on Java 11

```
org.jboss.arquillian.container.spi.client.container.LifecycleException: 
  The java process starting the managed server exited unexpectedly with code [1]

org.jboss.modules.ModuleNotFoundException: java.se
```

This exception usually means that `--add-modules=java.se` needs to be added to invoked process

**Comment by jmartisk** [comment]
Merged, this should be fixed in SR Metrics 1.1.3 and 2.0.0 (both are unreleased yet), if an upgrade is needed for WildFly, please file a WFLY jira

**Comment by rsvoboda** [comment]
Thanks, I tried also JDK 12 `OpenJDK Runtime Environment (build 12-ea+23)` and it works fine.
